lib/dotcom_web/components/system_status/commuter_rail_status.ex

```diff
@@ -0,0 +1,265 @@
+    other_alert_counts =
+      reject_cancellations_and_delays(alert_counts)
+
+    combine_alert_counts(other_alert_counts) ++
+      [{:alert, "#{cancellations + delays} Cancellations / Delays"}]
+  end
+
+  # We have at least one cancellation and no delays.
+  defp combine_alert_counts(%{cancellation: cancellations} = alert_counts) do
+    other_alert_counts = reject_cancellations_and_delays(alert_counts)
+    effect_string = if cancellations == 1, do: "Cancellation", else: "Cancellations"
+
+    combine_alert_counts(other_alert_counts) ++ [{:alert, "#{cancellations} #{effect_string}"}]
+  end
+
+  # We have at least one delay and no cancellations.
+  defp combine_alert_counts(%{delay: delays} = alert_counts) do
+    other_alert_counts = reject_cancellations_and_delays(alert_counts)
+    effect_string = if delays == 1, do: "Delay", else: "Delays"
+
+    combine_alert_counts(other_alert_counts) ++ [{:alert, "#{delays} #{effect_string}"}]
+  end
+
+  # The default case where we have non-cancellation and non-delay alerts.
+  # If there is one alert, we say "1 ...".
+  # If there are multiple alerts, but of only one type, we say "X ...".
+  # If there are multiple types of alerts, we combine them to just say "X Service Alerts".
+  defp combine_alert_counts(alert_counts) do
+    effect_count = alert_counts |> Map.keys() |> Enum.count()
+    total_count = Enum.reduce(alert_counts, 0, fn {_, count}, acc -> acc + count end)
+
+    case {effect_count, total_count} do
+      {0, _} ->
+        []
+
+      {1, 1} ->
+        effect = alert_counts |> Map.keys() |> List.first()
+        effect_string = effect |> Atom.to_string() |> Recase.to_title()
+
+        [{effect, "1 #{effect_string}"}]
+
+      {1, _} ->
+        effect = alert_counts |> Map.keys() |> List.first()
+        effect_string = effect |> Atom.to_string() |> Recase.to_title() |> Inflex.pluralize()
+        count = alert_counts[effect]
+
+        [{effect, "#{count} #{effect_string}"}]
+
+      _ ->
+        [{:alert, "#{total_count} Service Alerts"}]
+    end
+  end
+
+  # Rejects cancellations and delays from the alert counts
+  # so that we can separate cancellations and delays from other alerts.
+  defp reject_cancellations_and_delays(alert_counts) when alert_counts == %{}, do: %{}
+
+  defp reject_cancellations_and_delays(alert_counts) do
+    alert_counts
+    |> Enum.reject(fn {effect, _} -> Enum.member?(~w(cancellation delay)a, effect) end)
+    |> Map.new()
+  end
+
+  # A row that indicates that the service is not running today.
+  # This trumps any other status.
+  defp row(%{row: %{service_today?: false}} = assigns) do
+    ~H"""
+    <a
+      class={[
+        "flex items-center py-2",
+        "hover:bg-brand-primary-lightest cursor-pointer group/row",
+        "text-black no-underline font-normal",
+        "border-t-[1px] border-gray-lightest",
+        "min-h-12 text-md md:text-lg"
```

Comment:
```suggestion
        "min-h-12"
```

This was making the `No service today` text 18px on desktop, when it should be 16px.